README.md

```diff
@@ -25,13 +25,13 @@ Or install it yourself as:
 
 ```ruby
 require 'routemaster/client'
-client = Routemaster::Client.new(url: 'https://bus.example.com', uuid: 'demo')
+client = Routemaster::Client.new(url: 'https://bus.example.com', delivery_token: 'demo')
```

Comment:
So, I'm guessing I can start untangling the confusion here. You wouldn't connect to the bus with a "delivery token" as that's what the _bus_ uses to connect back to the _subscriber_. If you want to rename, I guess this could be called "client token".